File: medium/2390/2390. Removing Stars From a String.ts
function removeStars(s: string): string {
  let res = ''
  let count = 0
  const len = s.length
  for (let i = len - 1; i >= 0; i--) {
    if (count !== 0) {
      s[i] === '*' ? count++ : count--
      continue
    }
    if (s[i] === '*')
      count++
    else
      res += s[i]
  }
  return res.split('').reverse().join('')
}
removeStars('leet**cod*e') // 'abcde'

File: medium/2390/2390. Removing Stars From a String.py
class Solution:
    def removeStars(self, s: str) -> str:
      count = 0
      res = ''
      for i in range(len(s)-1,-1,-1):
        if count != 0:
          if s[i] != "*" :
            count -= 1
          else:
            count += 1
          continue
        if s[i] == "*":
          count += 1
        else:
          res += s[i]
      print(res)
      return res[::-1]
        
        
p = Solution()
print(p.removeStars("leet**cod*e"))
